site/gatsby-site/cypress/e2e/integration/discover.cy.js

```diff
@@ -257,7 +257,7 @@ describe('The Discover app', () => {
 
     cy.location('search', { timeout: 8000 }).should(
       'equal',
-      '?display=details&is_incident_report=true&page=1'
+      '?display=details&is_incident_report=true&page=1&sortBy=relevance'
```

Comment:
We should add another test to check if the sort selector is set with the value from the URL